docs/pages/careers.tsx

```diff
@@ -188,25 +188,20 @@ export default function Careers() {
           <Box data-mui-color-scheme="dark" sx={{ bgcolor: 'common.black' }}>
             <Section bg="transparent" cozy>
               <SectionHeadline
-                alwaysCenter
```

Comment:
to me this text feels awkward in the center of the page, but if others disagree then we can change it back.